packages/tools/client-debugger/client-debugger-chrome-extension/package.json

```
@@ -52,6 +52,8 @@
     "@types/chrome": "0.0.200",
     "@types/jest": "22.2.3",
     "@types/node": "^14.18.36",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
```

```suggestion
    "@types/react": "^17.0.44",
    "@types/react-dom": "^17.0.18",
```

So no one asks, "why are the versions here different than elsewhere in the client release group?"